dispersion_protocol/messages.md

```diff
@@ -0,0 +1,183 @@
+| estimator_status                | uint16_t   |       | [ESTIMATOR_STATUS_FLAGS](https://mavlink.io/en/messages/common.html#ESTIMATOR_STATUS_FLAGS) | Bitmap indicating which estimator outputs are valid.                                                                                                        |
+| landed_state                    | uint8_t    |       | invalid:MAV_LANDED_STATE_UNDEFINED MAV_LANDED_STATE                                         | The landed state. Is set to [MAV_LANDED_STATE_UNDEFINED](https://mavlink.io/en/messages/common.html#MAV_LANDED_STATE_UNDEFINED) if landed state is unknown. |
+| angular_velocity_z              | float      | rad/s | invalid:NaN                                                                                 | Z component of angular velocity in NED (North, East, Down). NaN if unknown.                                                                                 |
+
+> [!QUESTION]
+> This table is an exact clone of AUTOPILOT_STATE_FOR_GIMBAL_DEVICE because it felt like a good representation of the information a dispersion controller might need to deliver quality corrections. It at a minimum needs velocity. Can we just use something like GLOBAL_POS_INT or the GIMBAL message? I dont love the redundancy.
+
+# Enumerated Types
+
+## MAV_TYPE
+
+MAVLINK component type reported in HEARTBEAT message. Flight controllers must report the type of the vehicle on which they are mounted (e.g. MAV_TYPE_OCTOROTOR). All other components must report a value appropriate for their type (e.g. a camera must use MAV_TYPE_CAMERA).
+
+| Value | Name                                                      | Description                          |
+| :---- | :-------------------------------------------------------- | :----------------------------------- |
+| ...   | ...                                                       | ...                                  |
+| 45    | <span id="mav_type_dispersion">MAV_TYPE_DISPERSION</span> | A device that can disperse a payload |
+
+## MAV_COMPONENT
+
+Component ids (values) for the different types and instances of onboard hardware/software that might make up a MAVLink system (autopilot, cameras, servos, GPS systems, avoidance systems etc.).
+
+Components must use the appropriate ID in their source address when sending messages. Components can also use IDs to determine if they are the intended recipient of an incoming message. The MAV_COMP_ID_ALL value is used to indicate messages that must be processed by all components. When creating new entries, components that can have multiple instances (e.g. cameras, servos etc.) should be allocated sequential values. An appropriate number of values should be left free after these components to allow the number of instances to be expanded.
```

Comment:
Just FYI, (it does not make any difference to you) but in theory you can use any id you like, because the identity is supposed to come from the MAV_TYPE.

Assigning and using a  `MAV_COMP_ID_DISPERSION` is still good idea though, because it reduces the chance that by default the id is being used by something else. Also many systems will identify type on the component, even though that is "forbidden".